crates/bevy_reflect/Cargo.toml

```diff
@@ -42,6 +42,8 @@ smol_str = { version = "0.2.0", optional = true }
 ron = "0.8.0"
 rmp-serde = "1.1"
 bincode = "1.3"
+serde_json = "1.0"
+serde = { version = "1", features = ["derive"] }
```

Comment:
What was the compile error that required this addition?

Reply:
```
error: cannot find derive macro `Deserialize` in this scope
    --> crates/bevy_reflect/src/serde/de.rs:1165:41
     |
1165 |     #[derive(Reflect, Debug, PartialEq, Deserialize)]
     |                                         ^^^^^^^^^^^
     |
note: `Deserialize` is imported here, but it is only a trait, without a derive macro
    --> crates/bevy_reflect/src/serde/de.rs:1114:9
     |
1114 |     use serde::Deserialize
...
```

Reply:
Weird. I think CI should catch that if it's truly not compiling. But seeing as this is a dev-dependency, I suppose it doesn't hurt to add it.